packages/react-composites/src/composites/CallComposite/hooks/useHandlers.ts

```diff
@@ -110,6 +110,9 @@ const createCompositeHandlers = memoizeOne(
     /* @conditional-compile-remove(close-captions) */
     onStartCaptions: async (captionsOptions) => {
       await adapter.startCaptions(captionsOptions);
+      if (captionsOptions?.spokenLanguage && captionsOptions.spokenLanguage !== '') {
```

Comment:
This logic fix should be more like in the UI(composite) instead of handler,  because once we have the isCaptionGloballyActive in GA, it is also UI's choice to decide whether we show the modal or not.

also we might need to comment why we need a fix there in UI